### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Yonashenok**
## what I did
* check for linters error
* set up webpack
* use array of object to dynamically created list
* make sure webpack load CSS 
* all source file in /src directory

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  {
+    description: 'Adding a new item',
+    completed: false,
+    index: 6,
+  },
+];
+
+const renderToDoList = () => {
+  const markUp = toData
+    .map((toDo) => `
```
Comment (OyePriscilla) [inline src/index.js]: - [ ] Hello great developer! Awesome implementing most of the project requirement but kindly ensure that the list is ordered in accordance to the index values for each task. 👍 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Yonashenok 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used webpack to bundle JavaScript ✔️ 
Implemented an array of some simple to do tasks ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Yonashenok) [reply src/index.js]: Hey @OyePriscilla it in the order that why I put the index of object in the list

**Review by Shisui6 — APPROVED** [review]
Hi @Yonashenok, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Todo app setup properly ✅


## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**